tests/foreman/cli/test_puppetmodule.py

```diff
@@ -54,8 +55,7 @@ def test_puppet_module_info(self):
         return_value = PuppetModule.list({
             'organization-id': self.org['id'],
         })
-        # There are 4 puppet modules in the test puppet-module url
-        for i in range(4):
```

Comment:
We should not connect `_info` test with `.list` functionality as that code is already covered in previous test. Correspondingly, it doesn't make sense to block second test due that new BZ